css/style.css

```diff
@@ -0,0 +1,112 @@
+main {
+  margin: 30px auto;
+  width: 1024px;
+  box-shadow: 0px 0px 20px 1px rgba(0, 0, 0, 0.5);
+  padding: 10px 35px;
+  position: relative;
+}
+.main-info {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid black;
+  margin-bottom: 10px;
+  padding-bottom: 20px;
+}
+.main-info__title {
+  display: flex;
+  flex-direction: column;
+}
+.contacts__list{
```

Comment:
Follow same style in you code for spacing near brackets. In some places you have "class_name{" or "class_name {"